src/libtoast/src/toast_tod_filter.cpp

```diff
@@ -34,7 +34,7 @@ void toast::filter_polynomial(int64_t order, size_t n, uint8_t * flags,
     double fone = 1.0;
 
     #pragma \
-    omp parallel for schedule(static) default(none) shared(order, signals, flags, n, nsignal, starts, stops, nscan, norder, upper, lower, notrans, trans, fzero, fone)
```

Comment:
Maybe use default(shared)?

Reply:
I prefer not to use defaults. This way every shared variable is an explicit choice. Fewer surprises that way.

Reply:
But that is exactly what `default(none)` (now removed) does.  If no `default` is specified, then all unspecified variables are shared.  [One reference](https://rookiehpc.org/openmp/docs/default/index.html) and [another](https://www.openmp.org/spec-html/5.2/openmpsu35.html).

Reply:
Correct. I removed default(none) temporarily during development and forgot to add it back. It is now restored.